#
#	sigint.c:25: error: static declaration of 'sigint_reached_cb' follows non-static declaration
#	sigint.h:5: error: previous declaration of 'sigint_reached_cb' was here
#
#	main.c:50: error: static declaration of 'argp_program_version' follows non-static declaration
#	/usr/include/argp.h:423: error: previous declaration of 'argp_program_version' was here
#
#	cli/main.c      |    2 +-
#	sigint/sigint.h |    2 --
#	2 files changed, 1 insertion(+), 3 deletions(-)
#
--- poldek-0.19.0-cvs20041011.21/sigint/sigint.h.orig	2004-09-26 22:39:57.000000000 +0200
+++ poldek-0.19.0-cvs20041011.21/sigint/sigint.h	2004-12-14 15:31:53.387046032 +0100
@@ -2,8 +2,6 @@
 #ifndef POLDEK_SIGINT_H
 #define POLDEK_SIGINT_H
 
-extern void (*sigint_reached_cb)(void);
-
 void sigint_init(void);
 void sigint_destroy(void);
 void sigint_reset(void);
--- poldek-0.19.0-cvs20041011.21/cli/main.c.orig	2004-10-09 14:53:51.000000000 +0200
+++ poldek-0.19.0-cvs20041011.21/cli/main.c	2004-12-14 15:37:39.181477288 +0100
@@ -47,7 +47,7 @@
 
 extern int poclidek_shell(struct poclidek_ctx *cctx);
 
-static const char *argp_program_version = poldek_VERSION_BANNER;
+const char *argp_program_version = poldek_VERSION_BANNER;
 const char *argp_program_bug_address = poldek_BUG_MAILADDR;
 static char args_doc[] = N_("[PACKAGE...]");
 
